Gen3_Source_injection.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    agn_lenses = get_lens_radec(agn, butler, skymap=skymap)
</pre><p>perhaps these two lines and the next two blocks can get merged as they are executions of tasks to get the lenses and produce the coadds etc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/DC2-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>